copyright

```diff
@@ -736,11 +735,17 @@ License: CC0
 Comment: Taken from https://pxhere.com/en/photo/1071635 and cropped.
 
 Files:
- images/land/station15*
+ images/land/station7*
 Copyright: N/A (CC0 Public Domain)
 License: CC0
 Comment: Taken from https://pxhere.com/en/photo/119196 and cropped.
 
+Files:
+ images/land/station15*
+Copyright: N/A (CC0 Public Domain)
+License: CC0
+Comment: Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. According to Section 3 of the Pixabay Terms and Conditions, all images with a "published date" earlier than January 9, 2019 have the CC0 license.
```

Comment:
The page calls them "Terms of Service".
Should the last updated date be included, as well? It currently says April 4, 2023.
```suggestion
Comment: Taken from https://pixabay.com/photos/step-industry-steel-3104846/ and cropped. According to Section 3 of the Pixabay Terms of Service, all images with a "published date" earlier than January 9, 2019 have the CC0 license.
```